aim/agent/aid/universes/aci/converter.py

```diff
@@ -373,6 +373,75 @@ def l3ext_member_converter(object_dict, otype, helper,
     return result
 
 
+def bgp_extp_converter(object_dict, otype, helper,
+                       source_identity_attributes,
+                       destination_identity_attributes,
+                       to_aim=True):
+    result = []
+    if to_aim:
+        res_dict = {}
+        try:
+            id = default_identity_converter(object_dict, otype, helper,
+                                            to_aim=True)
+        except apic_client.DNManager.InvalidNameFormat:
+            return []
+        for index, attr in enumerate(destination_identity_attributes):
+            res_dict[attr] = id[index]
+        res_dict['bgp_enable'] = True
+        result.append(default_to_resource(res_dict, helper, to_aim=True))
+    else:
+        if (object_dict.get('bgp_enable')):
+            dn = default_identity_converter(object_dict, otype, helper,
+                                            aci_mo_type=helper['resource'],
+                                            to_aim=False)[0]
+            result.append({helper['resource']: {
+                'attributes': {'dn': dn}
+            }})
+    return result
+
+
+def bgp_as_converter(object_dict, otype, helper,
```

Comment:
why do we need this converter? Seems like the same result could be achieved by following the example of fvRsCtx

Reply:
For all comments on converter for bgpAs and LocalAs. I remember I had some issues initially. But I have changed the resource definition since then( previously this was under l3out interface). I will try without the converter again and see if it works.